spec/integration/spendings_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe 'Authentication', type: :request do
+  let(:user) { User.new(name: 'Bilal Ahmed', email: 'bilal@gmail.com', password: 'password') }
+
+  before do
+    sign_in user
+  end
+end
+
+RSpec.describe 'User', type: :feature do
+  let(:user) { FactoryBot.create(:user) }
+  let(:category) { FactoryBot.create(:category, user:) }
+  before do
+    sign_in user
+  end
+end
```

Comment:
- [ ] You have not written any integration tests here that test the functionality of your spendings. Please try to write robust integration tests here 🙏

Reply:
fixed:
![image](https://github.com/Ashnaali3255/Budget-App-rails/assets/116655378/c9100d53-e59e-453a-a286-e4915a58d41b)